sdcm/utils/alternator/data_generator.py

```diff
@@ -0,0 +1,179 @@
+import logging
```

Comment:
why we have all the random generator ? I thought we said we want specific data, and not randomely generated data for the upgrade test ?